### PR TITLE
UX: improved collapsed drawer state/interactions

### PR DESCRIPTION
**Pull request by jjaffeux**
- Clicking the channel title of a collapsed drawer will only open the drawer, and not open settings
- Remove the back button when the drawer is collapsed
- Uses same icon for toggling on chat that composer

### CODE REVIEW

File: app/assets/javascripts/discourse/app/components/composer-toggles.js [hunk 1]
```diff
@@ -29,13 +29,13 @@ export default class ComposerToggles extends Component {
...
       ? "xmark"
-      : "chevron-down";
+      : "angles-down";
   }
 
   @discourseComputed("composeState")
   fullscreenIcon(composeState) {
     return composeState === "draft"
-      ? "chevron-up"
+      ? "angles-up"
```
Comment (pmusaraj) [inline app/assets/javascripts/discourse/app/components/composer-toggles.js:38]: I would have picked the chevron as the default and used that in chat. Mostly from the perspective of opting to change the icon in the less travelled user path to match the current standard. Changing it in the composer is a much bigger "cheese move" for virtually all users (vs in chat, used by a smaller subset of users/instances).

File: plugins/chat/assets/stylesheets/common/chat-navbar.scss [hunk 2]
```diff
@@ -32,13 +32,43 @@
...
+    }
+
+    &__toggle-drawer-button {
+      &:after {
+        @container (inline-size) {
+          content: "";
+          position: absolute;
+          height: 100%;
+          left: 0;
+          width: calc(100cqw - 2.3em + 1rem);
```
Comment (pmusaraj) [inline plugins/chat/assets/stylesheets/common/chat-navbar.scss:50]: Wow, TIL: `1cqw` = 1% of the container's width. Nice!

**Review by pmusaraj — APPROVED** [review]
Looks good! The chat system failure is legit though.

File: app/assets/javascripts/discourse/app/components/composer-toggles.js (hunk 1 above)
Comment (jjaffeux) [reply app/assets/javascripts/discourse/app/components/composer-toggles.js:38]: This was the choice of chapoi in the internal discussion, you should rather say it here

**Pull request merged**